bigchaindb/commands/bigchaindb.py

```diff
@@ -143,6 +144,36 @@ def run_upsert_validator_new(args, bigchain):
         raise OperationError('Failed to commit election')
 
 
+def run_upsert_validator_approve(args, bigchain):
+    """Approve an election to add/update/remove a validator to an existing BigchainDB network
+
+    :param args: dict
+        args = {
+        'election_id': the election_id of the election (str)
+        'sk': the path to the private key of the signer (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: a success message
+    :raises: OperationError if the write transaction fails for any reason
+    """
+
+    key = load_node_key(args.sk)
+    tx = bigchain.get_transaction(args.election_id)
+    voting_power = ValidatorElection.get_validators(bigchain)[key.public_key]
+
+    approval = ValidatorElectionVote.generate(tx.to_inputs(), [
```

Comment:
The vote transaction cannot spend all the inputs, it can only spend the inputs which are allocated to the given user who is creating this vote.